src/components/ImageSlider/manifest.json

```diff
@@ -18,35 +18,69 @@
       "role": "listItem",
       "reference": "images"
     },
+    {
+      "name": "enableAutoplay",
+      "displayName": "Autoplay?",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "autoplayTime",
+      "displayName": "Autoplay Delay (in seconds)",
+      "type": "number",
+      "default": 3,
+      "control": {
+        "type": "slider",
+        "min": 0,
+        "max": 10
+      },
+      "enabled": {
+        "enableAutoplay": true
+      }
+    },
     {
       "name": "action",
       "displayName": "Click Actions",
       "type": "action",
       "role": "listItem",
       "reference": "images"
+    },
+    {
+      "name": "scrollAction",
+      "displayName": "Action when user moves to another image",
+      "type": "action",
+      "role": "listItem",
+      "reference": "images",
+      "arguments": [
+        {
+          "type": "number",
+          "displayName": "Next Image Number"
+        }
+      ]
     }
   ],
   "childComponents": [
     {
       "name": "dots",
-      "displayName": "Dots",
+      "displayName": "Slide Indicator",
       "props": [
         {
           "name": "enabled",
           "type": "boolean",
           "default": true
         },
         {
-          "name": "activeColor",
-          "displayName": "Active Color",
-          "type": "color",
-          "default": "@primary"
-        },
-        {
-          "name": "inactiveColor",
-          "displayName": "Inactive Color",
-          "type": "color",
-          "default": "#ddd"
+          "name": "showDots",
+          "displayName": "Show Dots or Numbers?",
```

Comment:
This copy sucks. We should do something like "Indicator Type"

Reply:
Also, this whole thing should be switched around so that dots are the default option. I had an app that previously showed dots and after loading these changes they now show numbers. By default it should show dots for old apps as well as new.